pycbc/population/rates_functions.py

```diff
@@ -66,17 +68,17 @@ def save_bkg_falloff(fname_statmap, fname_bank, path, rhomin, lo_tlt_mass, hi_tl
                Destination where txt file is saved
         rhomin: float
                Minimum value of SNR threhold (will need including ifar)
-        lo_tlt_mass: float
-               Low mass for template for trigger to be considered
-        hi_tlt_mass: float
-               High mass for template for trigger to be considered
+        lo_mchirp: float
+               Low chirp mass for the template
+        hi_mchirp: float
+               High chirp mass for template 
```

Comment:
Codacy shows a whitespace error on this line, make it happy by removing the space after 'template' !

Reply:
fixed all possible codacies